net/lighttpd/Makefile

```diff
@@ -159,25 +183,26 @@ $(eval $(call BuildPlugin,access,Access restrictions,,30))
 $(eval $(call BuildPlugin,accesslog,Access logging,,30))
 $(eval $(call BuildPlugin,alias,Directory alias,,30))
 $(eval $(call BuildPlugin,cgi,CGI,,30))
-#$(eval $(call BuildPlugin,cml,Cache Meta Language,+PACKAGE_lighttpd-mod-cml:liblua +PACKAGE_lighttpd-mod-cml:libmemcached,30))
-$(eval $(call BuildPlugin,cml,Cache Meta Language,+PACKAGE_lighttpd-mod-cml:liblua,30))
-$(eval $(call BuildPlugin,compress,Compress output,+PACKAGE_lighttpd-mod-compress:zlib,30))
+#$(eval $(call BuildPlugin,cml,Cache Meta Language,+PACKAGE_lighttpd-mod-cml:liblua5.3 +PACKAGE_lighttpd-mod-cml:libnettle +PACKAGE_lighttpd-mod-cml:libmemcached,30))
```

Comment:
I suggest getting rid of this comment.

Reply:
Some years ago, the maintainers had tried to build lighttpd with memcached, but gave up and move on so that other changes would proceed.

Should I try to get it working with memcached?

Reply:
I have removed the comment for mod_cml and for mod_trigger_b4_dl

Reply:
OpenWrt does not currently provide a libmemcached package.  I have removed those commented-out lines.